src/integrationTest/java/com/mongodb/kafka/connect/ConnectorValidationIntegrationTest.java

```diff
@@ -511,6 +627,14 @@ private boolean isReplicaSetOrSharded() {
     }
   }
 
+  private boolean isAtleastFiveDotZero() {
+    return getMongoClient()
+            .getDatabase("admin")
+            .runCommand(BsonDocument.parse("{isMaster: 1}"))
+            .get("maxWireVersion", 0)
+        >= 13;
```

Comment:
This method is different from `TimeseriesValidation.isAtleastFiveDotZero`. Maybe it is worth marking it public in `TimeseriesValidation` and reusing in tests?

Reply:
Agreed

Reply:
Done